<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Chatwidget</class>
 <widget class="QWidget" name="Chatwidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>749</width>
    <height>498</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>749</width>
    <height>498</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>749</width>
    <height>498</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>聊天窗口</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/picture/background1.jpg);</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_4">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="lbRoomname">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>房间名：</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="lbUsername">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>用户：</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="lbRoomuser">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>房主：</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QLabel" name="lbAccount">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>账户余额：</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="minimumSize">
          <size>
           <width>495</width>
           <height>31</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>495</width>
           <height>31</height>
          </size>
         </property>
         <property name="text">
          <string>聊天内容：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbVoice">
         <property name="minimumSize">
          <size>
           <width>31</width>
           <height>31</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>31</width>
           <height>31</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border-image: url(:/picture/mcallphone.png);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbVideo">
         <property name="maximumSize">
          <size>
           <width>31</width>
           <height>31</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border-image: url(:/picture/video.png);
</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTextBrowser" name="tbChat"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="pbDanmu">
           <property name="minimumSize">
            <size>
             <width>61</width>
             <height>23</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>61</width>
             <height>23</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(0, 170, 0);</string>
           </property>
           <property name="text">
            <string>弹幕</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbrecharge">
           <property name="minimumSize">
            <size>
             <width>61</width>
             <height>23</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>61</width>
             <height>23</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 0);</string>
           </property>
           <property name="text">
            <string>充值</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbGift">
           <property name="minimumSize">
            <size>
             <width>31</width>
             <height>21</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>31</width>
             <height>21</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">border-image: url(:/picture/liwu.jpg);</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLineEdit" name="leChat">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>101</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>101</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>当前用户：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="lwUser"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
